functions/Get-DbaInstallDate.ps1

```diff
@@ -0,0 +1,188 @@
+function Get-DbaInstallDate
+{
+<#
+.SYNOPSIS
+Returns the install date of SqlServer and Windows Server, depending on what is passed. 
+	
+.DESCRIPTION
+By default, this command returns for each SQL Server instance passed in:
+SQL Instance install date, formatted as a string
+Hosting Windows server install date, formatted as a string
+	
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER SqlCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER WindowsCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER SqlOnly
+Excludes the Windows server information
+
+.PARAMETER WindowsOnly
+Excludes the SQL server information
+
+.NOTES
+Tags: CIM 
+Original Author: Mitchell Hamann (@SirCaptainMitch), mitchellhamann.com
+	
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>
+
+.LINK
+https://dbatools.io/Get-DbaInstallDate
+
+.EXAMPLE
+Get-DbaInstallDate -SqlServer SqlBox1\Instance2
+
+Returns an object with SQL Server Install date as a string and the Windows install date as string. 
+
+.EXAMPLE
+Get-DbaInstallDate -SqlServer winserver\sqlexpress, sql2016
+
+Returns an object with SQL Server Install date as a string and the Windows install date as a string for both SQLInstances that are passed to the cmdlet.  
+	
+.EXAMPLE   
+Get-DbaInstallDate -SqlServer sqlserver2014a, sql2016 -SqlOnly
+
+Returns an object with only the SQL Server Install date as a string. 
+
+.EXAMPLE   
+Get-DbaInstallDate -SqlServer sqlserver2014a, sql2016 -WindowsOnly
+
+Returns an object with only the Windows Install date as a string. 
+
+.EXAMPLE   
+Get-SqlRegisteredServerName -SqlServer sql2014 | Get-DbaInstallDate
+
+Returns an object with SQL Server Install date as a string and the Windows install date as string for every server listed in the Central Management Server on sql2014
+	
+#>
+	[CmdletBinding(DefaultParameterSetName = "Default")]
+	Param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlInstance", "ComputerName")]
+		[object[]]$SqlServer,
+		[parameter(ParameterSetName = "Sql")]
+		[Switch]$SqlOnly,
+		[parameter(ParameterSetName = "Windows")]
+		[Switch]$WindowsOnly,
+		[Alias("Credential")]
+		[PsCredential]$SqlCredential,
+		[PsCredential]$WindowsCredential
```

Comment:
Also verify your parameters in help line up with the order of your parameters here.